app/models/admin/r_and_r_item.rb

```diff
@@ -0,0 +1,100 @@
+class Admin::RAndRItem < ApplicationRecord
+  belongs_to :digitization_queue_item, optional: true
+
+  before_destroy :remove_reference_from_dq_table
+
+  scope :open_status,   -> { where.not(status: "closed_r_and_r_request") }
+  scope :closed_status, -> { where(    status: "closed_r_and_r_request") }
+
+  # for now just validate bib numbers to not have the extra digit.
+  # We could try to 'automatically' fix them if this is still too
+  # confusing for curators
+  class StartsWithBValidator < ActiveModel::EachValidator
+    def validate_each(record, attribute, value)
+      value = record.send(attribute)
+      unless (value.blank? || value.starts_with?("b"))
+        record.errors[attribute] << (options[:message] || "must start with b")
+      end
+    end
+  end
+  validates :bib_number, length: { is: 8 }, allow_blank: true, starts_with_b: true
+
+  # collecting areas could have been normalized as a separate table, but
+  # not really needed, we'll just leave it as a controlled string.
+  COLLECTING_AREAS = %w{archives photographs rare_books modern_library museum_objects museum_fine_art}
+  validates :collecting_area, inclusion: { in: COLLECTING_AREAS }
+
+  STATUSES = %w{
+    awaiting_dig_on_cart imaging_in_process post_production_completed
+    files_sent_to_patron closed_r_and_r_request
+   }
+
+  CURATORS = %w{ ashley hillary jim patrick molly other }
```

Comment:
maybe just a text entry instead of a fixed list with 'other'?

Reply:
I will leave this as if for now, and prompt them when I demo.